pkg/virtctl/guestfs/guestfs_test.go

```diff
@@ -192,20 +192,17 @@ var _ = Describe("Guestfs shell", func() {
 
 		It("Image prefix from kubevirt config not discarded", func() {
 			guestfs.SetImageInfoGetFunc(fakeGetImageInfoNoCustomURL)
-			guestfs.ImageSetFunc(kubevirtClient)
-			Expect(*guestfs.ImagePtr).To(Equal("someregistry.io/kubevirt/some-prefix-libguestfs-tools@89af657d3c226ac3083a0986e19efe70c9ccd7e7278137e9df24b9b430182aa7"))
+			Expect(guestfs.ImageSetFunc(kubevirtClient)).Should(HaveValue(Equal("someregistry.io/kubevirt/some-prefix-libguestfs-tools@89af657d3c226ac3083a0986e19efe70c9ccd7e7278137e9df24b9b430182aa7")))
```

Comment:
Is `HaveEqual` really required?

Reply:
yes and no

yes because theoretically the return value of ImageSetFunc can be a nil ptr
so HaveValue checks this as well

on the UT we always use fake functions that do return a value, so in this specific case Equal can be enough

imho it is nicer and more robust to have it like this, but your call